storage/database/memory_database.go

```diff
@@ -48,56 +59,79 @@ func NewMemDBWithCap(size int) *MemDB {
 	}
 }
 
-func (db *MemDB) Type() DBType {
-	return MemoryDB
-}
-
-func (db *MemDB) Put(key []byte, value []byte) error {
+// Close deallocates the internal map and ensures any consecutive data access op
```

Comment:
op -> operation ?

Reply:
I also saw this-it is copied and pasted from geth code-but thought it is a kind of abbreviations.